cdap-ui/app/filters/number-with-ellipsis.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Similar to ellipsis.js filter, but show ellipses in the front instead
+ **/
+angular.module(PKG.name+'.filters').filter('myNumberWithEllipsis', function($filter) {
```

Comment:
I'm a little unclear on the difference between `myNumberWithEllipsis` and `myLeadingEllipsis` filters. What are the differences, and why are there two? The description you provided seems to be the same, to show ellipsis in the front